core/src/main/scala/dev/atedeg/ecscala/dsl/ECScalaDSL.scala

```diff
@@ -1,37 +1,167 @@
+ * val entity1 = world hasAn entity withComponents {
+ *       MyComponent1() &: MyComponent2() &: MyComponent3()
+ * }
+ * }}}
+ *
+ * '''Add components to an entity:'''
+ * {{{
+ *   *  entity1 + MyComponent()
+ *   *  entity1 withComponent MyComponent()
+ *   *  entity1 withComponents { MyComponent1() &: MyComponent2() }
+ * }}}
+ *
+ * '''Remove components from an entity:'''
+ * {{{
+ *   *  remove MyComponent() from entity1
+ *   *  entity1 - MyComponent()
+ *   *  remove { MyComponent1() &: MyComponent2() &: MyComponent3() } from entity1
+ * }}}
+ *
+ * '''Add a system to a world:'''
+ * {{{
+ * world hasA system[MyComponent &: CNil] { (_,_,_) => {}}
+ * }}}
+ *
+ * '''Get a view from a world:'''
+ * {{{
+ *   val view = getView[MyComponent1 &: MyComponent2 &: CNil] from world
+ * }}}
+ *
+ * '''Remove all entities and their components from a world:'''
+ * {{{
+ *   clearAll from world
+ * }}}
  */
-trait ECScalaDSL extends ExtensionMethodsDSL {
+trait ECScalaDSL extends ExtensionMethodsDSL with FromSyntax {
+
+  /**
+   * Keyword that enables the use of the word "entity" in the dsl.
+   */
   def entity: EntityWord = EntityWord()
+
+  /**
+   * Keyword that enables the use of the word "system" in the dsl.
+   */
+  def system[L <: CList](system: (Entity, L, DeltaTime, World, View[L]) => Deletable[L])(using
+      clt: CListTag[L],
+  )(using world: World): Unit = world.addSystem(system)(using clt)
+
+  /**
+   * Keyword that enables the use of the word "getView" in the dsl.
+   */
+  def getView[L <: CList](using clt: CListTag[L]) = ViewFromWorld(using clt)
+
+  /**
+   * Keyword that enables the use of the word "remove" in the dsl.
+   */
+  def remove(entity: Entity) = FromWorld(Seq(entity))
+
+  /**
+   * Keyword that enables the use of the word "remove" in the dsl.
+   */
+  def remove(entities: Seq[Entity]) = FromWorld(entities)
+
+  /**
+   * Keyword that enables the use of the word "remove" in the dsl.
+   */
+  def remove[L <: CList](componentsList: L)(using clt: CListTag[L]): FromEntity = FromEntity(componentsList)
+
+//  /**
+//   * Keyword that enables the use of the word "remove" in the dsl.
+//   */
+//  def remove[C <: Component](component: C)(using ct: ComponentTag[C]): FromEntity = FromEntity(component &: CNil)(using ct &: CNil)
+
+  /**
+   * Keyword that enables the use of the word "clearAll" in the dsl.
+   */
+  def clearAll: FromWorld = FromWorld(ClearWord())
 }
 
-private[dsl] case class ComponentWrapper() {
+private[dsl] trait FromSyntax {
+
+  class FromWorld(left: Seq[Entity] | ClearWord) {
+
+    /**
+     * This method enables the following syntax:
+     *
+     * {{{
+     *   * remove (entity1) from world
+     *   * clearAll from world
+     * }}}
+     */
+    def from(world: World): Unit = left match
+      case entitities: Seq[Entity] => entitities foreach { world.removeEntity(_) }
+      case _: ClearWord => world.clear()
+  }
+
+  class FromEntity(componentList: CList)(using clt: CListTag[? <: CList]) {
```

Comment:
Why are you passing a `CList` and not using a generic type `L <: CList` (like in `ViewFromWorld`)? It would also simplify the `clt` type:
```suggestion
  class FromEntity[L <: CList](componentList: L)(using clt: CListTag[L]) {
```